.storybook/preview.js

```diff
@@ -1,8 +1,18 @@
-import { themes } from "@storybook/theming";
 import "vanilla-framework/scss/build.scss";
 
+import { themes } from "@storybook/theming";
+import { Title, Description, ArgTypes, Stories } from '@storybook/blocks';
+
 export const parameters = {
   docs: {
     theme: themes.vanillaish,
+    page: () => (
+      <>
+        <Title />
+        <Description />
+        <ArgTypes sort="alpha" />
```

Comment:
the custom page is here to keep the same layout that is being used today, otherwise the layout adds a `<Primary />` example, and instead of `<Argtypes />` it uses `<Controls />`

### Without custom page

<img width="1276" alt="image" src="https://github.com/canonical/react-components/assets/16295402/09c2b76e-bf11-4921-ab81-7541737e5856">


### With custom page

<img width="1275" alt="image" src="https://github.com/canonical/react-components/assets/16295402/89597704-8de6-41a2-a2fd-57745a5d5900">

Reply:
I don't think we are strongly attached to current page layout. If CSF defaults are different (with primary example on top) I think we better stick to defaults first, and then see if we want to change anything.

Reply:
agree, I believe the objective here is to lower cognitive overload when giving maintenance

changed to default layout on [1820f44](https://github.com/canonical/react-components/pull/1010/commits/1820f44963ec35b5032e970bf2190b2b4cbfcf8d)